src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+public class Tagging {
+	Person person;
+	Tag tag;
+	boolean isAdd;
```

Comment:
A boolean variable works here, but an `enum` may be more appropriate because the two values we need are `added` and `deleted`, not `true` and `false`.